server/controllers/events/events_controller_e2e_test.go

```diff
@@ -1124,19 +1140,18 @@ func setupE2E(t *testing.T, repoDir string, opt setupOption) (events_controllers
 			TerraformExecutor: terraformClient,
 			DefaultTFVersion:  defaultTFVersion,
 		},
-		PlanStepRunner: &runtime.PlanStepRunner{
-			TerraformExecutor: terraformClient,
-			DefaultTFVersion:  defaultTFVersion,
-		},
+		PlanStepRunner: runtime.NewPlanStepRunner(
```

Comment:
Why no ampersand?
```suggestion
		PlanStepRunner: &runtime.NewPlanStepRunner(
```

Reply:
`runtime.NewPlanStepRunner` returns `Runner` which is interface as a pointer. so not need it.